com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/util/FileUtils.java

```diff
@@ -24,6 +27,19 @@ public static void copyFile(Path fromPath, Path toPath) throws IOException {
         }
         Files.copy(fromPath, toPath, StandardCopyOption.REPLACE_EXISTING, StandardCopyOption.COPY_ATTRIBUTES);
     }
+    
+    public static List<String> splitFilesInChunks(List<String> fileIds) {
```

Comment:
Why this method is in FileUtils? The class here is supposed to work only with files. This method is working with strings which are named fileIds :) and it is splitting them if they contain "," otherwise it is taking them whole.
Maybe extracting it in some other util class is a good idea, what do you think? :)

Reply:
Good remark. Moved to NameUtil.